roles/deploy/defaults/main.yml

```diff
@@ -79,7 +77,13 @@ project_environment: {}
 #   - cmd: wp db import posts.sql
 project_pre_build_commands_local: []
```

Comment:
There's another `- path: ../example.dev/...` just above in the comments, to be replaced with the new `- path: {{ project.local_path }}/...`